front-end/src/app/shared/services/fec-api.service.ts

```diff
@@ -36,14 +39,10 @@ export class FecApiService {
     if (!committeeId) {
       throw new Error('No Committee Id provided.');
     }
-    const headers = this.getHeaders();
```

Comment:
Weren't we planning on making this configurable.  We don't want to eliminate the actual code that works, but point to something else when it's configured to mock right?

Reply:
made changes per our dev meeting for the new approach to overriding committee data in the api.  I believe this is OBE now.